tools/vargo/src/main.rs

```diff
@@ -447,7 +449,7 @@ fn run() -> Result<(), String> {
         std::path::PathBuf::from("target").join(if release { "release" } else { "debug" });
 
     let dashdash_pos =
-        (!in_nextest && (cmd == "test" || cmd == "nextest" || cmd == "fmt")).then(|| {
+        (!in_nextest && matches!(task, Task::Test { nextest: _ } | Task::Fmt)).then(|| {
```

Comment:
Minor stylistic choice; feel free to ignore this suggestion (I can see reasons to pick either of the two options tbqh):
```suggestion
        (!in_nextest && matches!(task, Task::Test { .. } | Task::Fmt)).then(|| {
```